plugin/src/main/java/org/opensearch/ml/action/model_group/TransportRegisterModelGroupAction.java

```diff
@@ -154,6 +154,9 @@ private void validateRequestForAccessControl(MLRegisterModelGroupInput input, Us
             } else {
                 input.setModelAccessMode(ModelAccessMode.RESTRICTED);
             }
+            if (modelAccessControlHelper.isUserHasAllPassedBackendRoles(user, input.getBackendRoles())) {
```

Comment:
Should this be negation? Also we can reuse isUserHasBackendRole() method instead of this new one

Reply:
Good catch, the naming convention for this method is not proper, also the admin could be blocked by this.